docs/dev/libvirt/README.md

```diff
@@ -247,6 +247,31 @@ This step allows installer and users to resolve cluster-internal hostnames from
 
 3. Reload NetworkManager to pick up the `dns` configuration change: `sudo systemctl reload NetworkManager`
 
+### Setting the resolver on Fedora 34+ with systemd-resolved
+
+Fedora 34 switched to utilizing systemd-resolved and changes how DNS resolution works on a per-interface basis. 
+
+If you are running Fedora 34, you must indicate that you want your resolver to point to the local dnsmasq resolver on the `tt0` interface.
```

Comment:
I find this sentence a bit confusing, we want to configure systemd-resolved to use the dnsmasq resolver we configured before for the `tt0` interface rather than the one libvirt set up automatically for this interface.

Reply:
Noted, I will rephrase this to make it clearer as to what is actually happening.